### ISSUE TITLE
[FR] Is it possible to support rectangular areas in menu?

### ISSUE THREAD

**Issue by math-feng** [RyotaUshio/obsidian-pdf-plus#81]
### Describe your request

Now select the text in the PDF, and then right-click the pop-up menu you can choose to specify the format to copy the link. Is it possible to support rectangular areas? The final effect is like the previous embedded PDF, but also to support their own adjustment of the size of the border area.

**Comment by RyotaUshio** [comment]
That's actually on the roadmap! Just to clarify, when you copy a link to a text selection, you want something like

```
[[file.pdf#page=...&selection=...&rect=...]]
```

to be copied to the clipboard, right?

**Comment by math-feng** [comment]
No, it's adding an option to this menu for a rectangular localized area
![image](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/78469397/f061e511-f3a6-46f2-8d66-65b6cde30dac)
